src/main/java/com/garbagemule/MobArena/signs/RendersTemplate.java

```diff
@@ -66,6 +80,25 @@ private String joining(String line, Arena arena) {
             .replace("<ready-players>", valueOf(arena.getReadyPlayersInLobby().size()));
     }
 
+    private String replaceReady(String line, Arena arena) {
+        Matcher matcher = readyPattern.matcher(line);
+        if (matcher.find()) {
+            int i = Integer.parseInt(matcher.group(2));
+            if (i > 0) {
+                if (matcher.group(1) == null) {
+                    // Variable is ready i
+                    List<Player> ready = arena.getReadyPlayers();
+                    return matcher.replaceFirst((ready.size()>=i) ? ready.get(i-1).getName() : "");
```

Comment:
I feel this (and the notready instance below) is unnecessarily dense. I don't like the choice operator when it results in "long lines". While more verbose, I do think this is cleaner:

```
if (ready.size() < i) {
    return matcher.replaceFirst("");
}

String name = ready.get(i - 1);
return matcher.replaceFirst(name);
```